pom.xml

```diff
@@ -58,6 +58,7 @@
     <version.com.github.tdomzal.junit.docker>0.3</version.com.github.tdomzal.junit.docker>
     <version.com.spotify.docker.client>5.0.2</version.com.spotify.docker.client>
     <version.org.apache.logging.log4j>2.8.2</version.org.apache.logging.log4j>
+    <version.org.glassfish.jersey.inject>2.29.1</version.org.glassfish.jersey.inject>
```

Comment:
@mareknovotny I wonder if these properties shouldn't be in kie-parent. Wdyt?

Reply:
if they are not overriding any version property in KIE parent, they should be declared in KIE PARENT

Reply:
@mareknovotny Adding this version to `kie-parent` should be in a separate PR, since this PR is responsible for upgrading the Spring version in `kie-parent` and not the `glassfish.jersey.inject` version. I would prefer to introduce/upgrade different dependency versions in separate PRs. WDYT?

Reply:
Yes, @cuijulian makes sense, I just asked question above because it struck my mind.